tests/unit/spec/base/mozilla-traffic-cop.js

```diff
@@ -118,6 +118,81 @@ describe('mozilla-traffic-cop.js', function () {
 
             expect(cop.verifyConfig()).toBeFalsy();
         });
+
+        it('should return true with a valid cookieExpires value', function() {
+            var cop = new Mozilla.TrafficCop({
+                id: 'test123',
+                cookieExpires: 48,
+                variations: {
+                    'v=1': 20,
+                    'v=2': 40
+                }
+            });
+
+            expect(cop.verifyConfig()).toBeTruthy();
+        });
+
+        it('should return false with an invalid cookieExpires value', function() {
+            var config1;
+            var config2;
+            var config3;
+
+            config1 = config2 = config3 = {
+                id: 'test123',
+                variations: {
+                    'v=1': 20,
+                    'v=2': 40
+                }
+            };
+
+            config1.cookieExpires = '2 days';
+            config2.cookieExpires = null;
+            config3.cookieExpires = [];
+
+            var cop = new Mozilla.TrafficCop(config1);
+            var cop2 = new Mozilla.TrafficCop(config2);
+            var cop3 = new Mozilla.TrafficCop(config3);
+
+            expect(cop.verifyConfig()).toBeFalsy();
+            expect(cop2.verifyConfig()).toBeFalsy();
+            expect(cop3.verifyConfig()).toBeFalsy();
+        });
+    });
+
+    describe('Mozilla.TrafficCop.cookieExpiresDate', function() {
+        var config = {
+            id: 'test123',
+            variations: {
+                'v=1': 40,
+                'v=2': 30
+            }
+        };
+
+        it('should return a date 24 hours into the future when no hours are specified', function() {
+            var cop = new Mozilla.TrafficCop(config);
+            var expiry = cop.cookieExpiresDate(new Date(2017, 0, 1, 12, 30));
+            expect(expiry).not.toBe(null);
+            expect(expiry.getFullYear()).toBe(2017);
+            expect(expiry.getMonth()).toBe(0);
+            expect(expiry.getDate()).toBe(2); // 24 hours default cookie length
+        });
+
+        it('should return an appropriate future date when hours are specified', function() {
+            config.cookieExpires = 72;
+            var cop = new Mozilla.TrafficCop(config);
+            var expiry = cop.cookieExpiresDate(new Date(2017, 0, 1, 12, 30));
+            expect(expiry).not.toBe(null);
+            expect(Object.prototype.toString.call(expiry)).toEqual('[object Date]');
```

Comment:
Heh, yeah, I was writing in implementation mode instead of testing mode. Removed. 👍